modules/core/src/lib/layer.js

```diff
@@ -599,12 +559,6 @@ export default class Layer extends Component {
     });
 
     this._updateState();
-
-    const model = this.getSingleModel();
```

Comment:
Why this could be removed?

Reply:
- Model id does not serve any purpose other than for debugging
- All models created by build-in layers already have ids